connector/mqtt/vernemq/v2k-bridge/app/MQTTClient.js

```diff
@@ -93,39 +108,40 @@ class MQTTClient {
   /**
    * Reached when the MQTTClient disconnects from the broker.
    *
-   * @callback MQTTClient~onDisconnect
+   * @function onDisconnect
+   * @private
    */
   onDisconnect() {
-    this.logger.info(`Client ${this.mqttOptions.clientId} disconnected, reconnecting...`);
+    this.logger.warn(`Client ${this.mqttOptions.clientId} disconnected, reconnecting...`);
     this.isConnected = false;
-    this.mqttc.reconnect();
```

Comment:
When the MQTT broker restarts, does the v2k service reconnects automatically?

Reply:
Yes, we do not need to explicitly call the reconnect function. There is the `reconnectPeriod` option in the (MQTT Client)[https://www.npmjs.com/package/mqtt#client] that enables the reconnection.